wc/src/wc.yml

```diff
@@ -0,0 +1,30 @@
+name: echo
+version: "0.0.0"
+author: Giacomo Parolini <silverweed1991@gmail.com>
+about: "Print newline, word and byte counts for each FILE, and a total line if more than one FILE is specified. A word is a non-zero-length sequence of characters delimited by white space.\n\nWith no FILE, or when FILE is -, read standard input."
```

Comment:
Use the word `display` over `print`

Reply:
Ok. Just in the "about" or also in all the flags help section?